src/jdk.incubator.vector/share/classes/jdk/incubator/vector/VectorOperators.java

```diff
@@ -452,6 +452,10 @@ static boolean opKind(Operator op, int bit) {
     public static final Unary NEG = unary("NEG", "-a", VectorSupport.VECTOR_OP_NEG, VO_ALL|VO_SPECIAL);
     /** Produce {@code bitCount(a)} */
     public static final Unary BIT_COUNT = unary("BIT_COUNT", "bitCount", VectorSupport.VECTOR_OP_BIT_COUNT, VO_NOFP);
+    /** Produce {@code numberOfTrailingZeros(a)} */
+    public static final Unary CTZ = unary("CTZ", "numberOfTrailingZeros", VectorSupport.VECTOR_OP_CTZ, VO_NOFP);
```

Comment:
I think we should be more verbose in the names, as they will not be well understood by developers. Specifically `TRAILING_ZEROS_COUNT` and `LEADING_ZEROS_COUNT`, and be consistent in other places. Internally we could shorten to `VECTOR_OP_TZ_COUNT` etc. to reduce the horizontal width.